src/lib/SX127xDriver/SX127x.cpp

```diff
@@ -321,7 +335,11 @@ bool SX127xDriver::DetectChip(SX12XX_Radio_Number_t radioNumber)
   {
     uint8_t version = hal.readRegister(SX127X_REG_VERSION, radioNumber);
     DBG("%x", version);
+    #if defined(RADIO_SX1272)
+    if (version == 0x22)
```

Comment:
If we make version a define in SX127xRegs.h we can remove these #if def and minimise some code.

Reply:
done